js/app.js

```diff
@@ -1 +1,157 @@
-// Your code here
+//MODEL
+var model = {
+  loggedIn: false,
+  userId: '',
+  username: '',
+  categories: [],
+  articles: []
+};
+
+
+//VIEW
+var loginTemplate;
+var categoryTemplate;
+var articleTemplate;
+
+function compileTemplates() {
+  var loginSource = $('#login-template').html();
+  loginTemplate = Handlebars.compile(loginSource);
+
+  var categorySource = $('#categories-template').html();
+  categoryTemplate = Handlebars.compile(categorySource);
+
+  var articleSource = $('#articles-template').html();
+  articleTemplate = Handlebars.compile(articleSource);
+}
+
+function renderLogin() {
+	var display = loginTemplate(model);
+	$('#loginContainer').html(display);
+}
+
+function renderCategories() {
+	$('#categories').html('');
+ 	for (var i = 0; i < model.categories.length; i++) {
```

Comment:
Nice, most people used the loop in the template - I like that you tried it this way.